src/platform/monitoring/DowntimeNotification/README.md

```diff
@@ -0,0 +1,19 @@
+# Downtime Notifications
+Downtime Notications provide a method of handling service outages in front-end applications. For example, if your application _My App_ leveraged a service called _Fancy Service_, and _Fancy Service_ was known to be offline for maintenance every now and then, it would be desirable to show a message to the user explaining that _My App_ is unavailable at this time so that users aren't given the impression that they are running into unexpected errors and so that _Fancy Service_ isn't needlessly hit. This should also be easy for engineers to do, without requiring any deployments to publish/unpublish the messaging.
```

Comment:
we could break this down into a list with some scannable headings. example:

```
## Overview  
- Displays a notification to users when a service dependent application is unavailable 
- Configuration driven- does not require an application build? 
  // (not really sure how this feature works- I assume we deploy a configuration change somehow- 
  // either way emphasize how it differs from our build / deploy process since that's the most pertinent information) 
  // how we describe the deploy process: https://department-of-veterans-affairs.github.io/veteran-facing-services-tools/getting-started/workflow/deploy#overview

_For example, if your application _My App_ leveraged a service called _Fancy Service_, and _Fancy Service_ was known to be offline for maintenance every now and then, it would be desirable to show a message to the user explaining that _My App_ is unavailable at this time_...
```

- 2nd person is good
- The [documentation style guide](https://department-of-veterans-affairs.github.io/veteran-facing-services-tools/documentation-guide/documentation-style-guide) might be helpful. it's also open to changes. in this case I would recommend simplifying some of the sentences so they're easier to scan.